### PR TITLE
Documentation for fosscord mail

### PR DESCRIPTION
**Pull request by MaddyUnderStars**
https://github.com/fosscord/fosscord-server/pull/965

### CODE REVIEW

File: docs/setup/server/email.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    | -------------------- | ------ | --------------- |
+    | email_mailgun_apiKey | string | Mailgun API key |
+    | email_mailgun_domain | string | Mailgun domain  |
+
+=== "Mailjet"
+
+    | key                     | type   | description        |
+    | ----------------------- | ------ | ------------------ |
+    | email_mailjet_apiKey    | string | Mailjet API key    |
+    | email_mailjey_apiSecret | string | Mailjet API secret |
```
Comment (TheArcaneBrony) [inline docs/setup/server/email.md:55]: Mailjey?

File: docs/setup/server/email.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+| ------------------------ | --------------------------------------------------------- |
+| `{instanceName}`         | `general_instanceName` config value                       |
+| `{userUsername}`         | The username of the user this email is being sent to      |
+| `{userDiscriminator}`    | The discriminator of the user this email is being sent to |
+| `{userId}`               | The ID of the user this email is being sent to            |
+| `{phoneNumber}`          | The last 4 digits of the user's phone number.             |
+| `{userEmail}`            | The user's email address                                  |
+| `{emailVerificationUrl}` | The generated email verification URL                      |
+| `{passwordResetUrl}`     | The generated password reset URL                          |
+| `{ipAddress}`            | The IP address that requested this email                  |
```
Comment (Puyodead1) [inline docs/setup/server/email.md:80]: This is incorrect, the IP/location related placeholders are for the "New location detected" emails (which are currently unimplemented). However, including the IP/location of the user that triggered the email could be added.

**Review by Puyodead1 — CHANGES_REQUESTED** [review]
also, mention that the `general_correspondenceEmail` configuration setting needs to be set to the "from" address

File: docs/setup/server/email.md (hunk 1 above)
Comment (Puyodead1) [reply docs/setup/server/email.md:55]: lol, I made that typo all the time when I was writing the server PR

**Review by Puyodead1 — APPROVED** [review]

**Pull request merged**